pg_search/src/postgres/customscan/builders/custom_path.rs

```diff
@@ -307,6 +307,25 @@ impl<P: Into<*mut pg_sys::List> + Default> CustomPathBuilder<P> {
         self
     }
 
+    pub fn set_already_presorted(mut self, presorted: bool) -> Self {
+        // According to PostgreSQL source code in src/include/nodes/pathnodes.h,
+        // For CustomPath nodes, flags is stored on the CustomPath struct directly,
+        // not on the Path struct
+        // #define CUSTOMPATH_PRESORTED 0x0001
+
+        // This flag tells PostgreSQL's optimizer that this path delivers tuples
+        // that are already presorted according to the pathkeys we've added.
+        // When only a partial set of pathkeys are satisfied, this will trigger
+        // the Incremental Sort node rather than a full Sort.
+        if presorted {
+            // Set flag on the CustomPath struct
+            self.custom_path_node.flags |= 0x0001; // CUSTOMPATH_PRESORTED
```

Comment:
Where does `0x0001` come from in the Postgres sources?  I can't find anything named `CUSTOMPATH_PRESORTED` or similar.

I know the comment says `src/include/nodes/pathnodes.h`, but I don't see that macro in the pg16 or 17 sources, so I'm a bit confused?

Reply:
Oops! That wasn't supposed to be checked in. I was toying with PG sources to see if `flags` can have any impact on sorting functionality from a custom scan. It's removed now.